BitwardenShared/UI/Platform/Application/Appearance/StyleGuideFont.swift

```diff
@@ -112,28 +112,49 @@ extension View {
     ///
     /// - Parameters:
     ///   - style: The style of the text.
+    ///   - includeLinePadding: A flag to indicate if the style should apply padding around the
+    ///     view to account for the font's line height. Defaults to `true`.
```

Comment:
The style guide modifier adds extra padding to text to try and match the line height of the font in Figma. But this was causing some issues when trying to align the views at the top of the action card, where the text was too low compared to the badge and close button.

![Simulator Screenshot - iPhone 15 Pro - 2024-09-27 at 16 47 00](https://github.com/user-attachments/assets/036e7350-1bf7-4be7-b92f-df601d5260ad)